Task/Taskfile.yml

```diff
@@ -239,3 +239,45 @@ tasks:
       - find {{.ROOT_DIR}} -type f -name '*.pyc' -delete
       - find {{.ROOT_DIR}} -type d -name '.ruff_cache' -exec rm -rf {} +
       - find {{.ROOT_DIR}} -type d -name '.task' -exec rm -rf {} +
+      - find {{.ROOT_DIR}} -type f -name 'sbom.*.json' -delete
+      - find {{.ROOT_DIR}} -type f -name 'vulns.*.json' -delete
+
+  sbom:
+    desc: Generate project SBOMs
+    dir: ../../..
+    preconditions:
+      - sh: which syft
+        msg: "Syft must be installed and reasonably current"
+    vars:
+      IMAGE_AND_TAG: '{{.IMAGE_NAME}}:{{.VERSION}}'
+      PLATFORM: '{{if eq .PLATFORM "all"}}{{.SUPPORTED_PLATFORMS}}{{else if .PLATFORM}}{{.PLATFORM}}{{else}}{{.LOCAL_PLATFORM}}{{end}}'
+    cmds:
+      - for:
```

Comment:
This is a neat trick, recently introduced to `Taskfile`s and very useful so we don't need to over-manage the syntax we are forced into due to `docker buildx`'s `--platform` comma separated platforms.  We just split on it and make it a local `platform` var, then transform any `/`s in the platform(s) to `_`s for filenames, otherwise the `/` will make folders.